openedx/core/lib/xblock_builtin/xblock_capa/xblock_capa/__init__.py

```diff
@@ -218,12 +225,19 @@ def add_resource_urls(self, fragment):
         for js_file in self.js_dependencies():
             fragment.add_javascript_url(staticfiles_storage.url(js_file))
 
+    @property
+    def loader(self):  # pylint: disable=no-self-use
+        """
+        Loader for loading and rendering assets stored in child XBlock package
```

Comment:
Please say "stored in this XBlock's package". This CAPA XBlock does not have any child XBlocks.